src/daft-core/src/array/ops/list.rs

```diff
@@ -625,6 +628,49 @@ impl ListArray {
             self.validity().cloned(),
         ))
     }
+
+    pub fn list_contains(&self, contains: &Series) -> DaftResult<BooleanArray> {
+        let seed = UInt64Array::from_iter(
+            Field::new("seed", DataType::UInt64),
+            std::iter::repeat(u64::MAX).map(Some).take(1),
+        )
+        .into_series();
+        macro_rules! gen_seed {
+            ($size:expr) => {
+                Some(seed.broadcast($size)?.u64()?)
+            };
+        }
```

Comment:
What's the purpose of having a seed and why a macro?

Reply:
I believe if we don't seed the two different Series, two different random seeds would be used and the hashes would turn out different, even if the underlying datas are equivalent.

I used the macro to remove some of the boilerplate, nothing too special over there.